tables/da-dk-g16-lit.ctb

```diff
@@ -1,6 +1,6 @@
 # Liblouis: Danish table for 6 dots grade 1 forward translation (literary)
 #
-#  Copyright (C) 2014-2017, Bue Vester-Andersen <bue@vester-andersen.dk>
+#  Copyright (C) 2014-2020, Bue Vester-Andersen <bue@vester-andersen.dk>
```

Comment:
Normally you should only use this format if every year in the range really is a copyrightable year that would be listed individually.

Reply:
> Ok, what does one usually do then? Just right the current year, i.e. 2020?

Comma separated list.

Reply:
> I guess I will just keep 2014-2020

OK, fine.